tests/system-tests/ran-du/tests/stability-workload.go

```diff
@@ -0,0 +1,136 @@
+package ran_du_system_test
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/namespace"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/await"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/platform"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/shell"
+	. "github.com/openshift-kni/eco-gotests/tests/system-tests/internal/stability"
+	. "github.com/openshift-kni/eco-gotests/tests/system-tests/ran-du/internal/randuinittools"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/ran-du/internal/randuparams"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/ran-du/internal/randutestworkload"
+)
+
+var _ = Describe(
+	"StabilityWorkload",
+	Ordered,
+	ContinueOnFailure,
+	Label("StabilityWorkload"), func() {
+		var (
+			clusterName string
+		)
+		BeforeAll(func() {
+			By("Preparing workload")
+			if namespace.NewBuilder(APIClient, RanDuTestConfig.TestWorkload.Namespace).Exists() {
+				By("Deleting workload using shell method")
+				_, err := shell.ExecuteCmd(RanDuTestConfig.TestWorkload.DeleteShellCmd)
+				Expect(err).ToNot(HaveOccurred(), "Failed to delete workload")
+			}
+
+			if RanDuTestConfig.TestWorkload.CreateMethod == randuparams.TestWorkloadShellLaunchMethod {
+				By("Launching workload using shell method")
+				_, err := shell.ExecuteCmd(RanDuTestConfig.TestWorkload.CreateShellCmd)
+				Expect(err).ToNot(HaveOccurred(), "Failed to launch workload")
+			}
+
+			By("Waiting for deployment replicas to become ready")
+			_, err := await.WaitUntilAllDeploymentsReady(APIClient, RanDuTestConfig.TestWorkload.Namespace,
+				randuparams.DefaultTimeout)
+			Expect(err).ToNot(HaveOccurred(), "error while waiting for deployment to become ready")
+
+			By("Waiting for statefulset replicas to become ready")
+			_, err = await.WaitUntilAllStatefulSetsReady(APIClient, RanDuTestConfig.TestWorkload.Namespace,
+				randuparams.DefaultTimeout)
+			Expect(err).ToNot(HaveOccurred(), "error while waiting for statefulsets to become ready")
+
+			By("Waiting for pods replicas to become ready")
+			_, err = await.WaitUntilAllPodsReady(APIClient, RanDuTestConfig.TestWorkload.Namespace, randuparams.DefaultTimeout)
+			Expect(err).ToNot(HaveOccurred(), "pod not ready: %s", err)
+
+			By("Fetching Cluster name")
+			clusterName, err = platform.GetOCPClusterName(APIClient)
+			Expect(err).ToNot(HaveOccurred(), "Failed to get cluster name")
+
+		})
+		It("StabilityWorkload", reportxml.ID("TBD"), Label("StabilityWorkload"), func() {
+
+			outputDir := RanDuTestConfig.StabilityOutputPath
```

Comment:
Is there a convention on which output path can be used to store output files?

Reply:
We don't have any convention but I think `/tmp` is good enough for this.

Reply:
I finally set it in by default in /tmp/resports